config/environment.js

```diff
@@ -25,6 +25,19 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+    torii: {
+      disableRedirectInitializer: true,
+      providers: {
+        'acmidm-oauth2': {
+          apiKey: '677af572-2e2f-4e61-ad38-036723bb314c',
```

Comment:
Do these values need to be checked in? I think just using the placeholders here would be better?

That seems to be how GN does it:
https://github.com/lblod/frontend-gelinkt-notuleren/blob/e344376bda3b8aa276ec29407fb7a5842b3160cc/config/environment.js#L56-L67

Reply:
yes probably, I did it this way  so we can test using ember serve, but anyway i don't think that will work (thus using mock login in that case). i'll change it